utils/web3.ts

```diff
@@ -217,25 +219,21 @@ export const checkInterface = async (address: string, web3: Web3) => {
   let isLsp0Erc725Account = false;
   try {
     isLsp0Erc725Account = await Contract.methods
-      .supportsInterface(LSP_INTERFACE_IDS.LSP0ERC725Account)
+      .supportsInterface(INTERFACE_IDS.LSP0ERC725Account)
       .call();
+    if (!isLsp0Erc725Account) {
+      isLsp0Erc725Account = await Contract.methods
+        .supportsInterface(LEGACY_LSP0_ACCOUNT_INTERFACE_ID)
```

Comment:
we might just want to drop support of old contracts

Reply:
Yeah maybe once we get rid of L14 we can remove these legacy contracts. It can be useful to have it for debugging.